src/rp2_common/hardware_clocks/include/hardware/clocks.h

```diff
@@ -173,8 +173,23 @@ void clocks_enable_resus(resus_callback_t resus_callback);
  * \param gpio The GPIO pin to output the clock to. Valid GPIOs are: 21, 23, 24, 25. These GPIOs are connected to the GPOUT0-3 clock generators.
  * \param src  The source clock. See the register field CLOCKS_CLK_GPOUT0_CTRL_AUXSRC for a full list. The list is the same for each GPOUT clock generator.
  * \param div  The amount to divide the source clock by. This is useful to not overwhelm the GPIO pin with a fast clock.
```

Comment:
prefer changing the names to `div_int` and `div_frac` and updating the doxygen to be a bit more specific about integer, and fraction between 0 and 255